src/Keboola/StorageApi/Components.php

```diff
@@ -326,4 +328,27 @@ public function createConfigurationWorkspace($componentId, $configurationId, arr
             $options
         );
     }
+
+    public function addConfigurationMetadata(ConfigurationMetadata $options)
+    {
+        return $this->client->apiPost(
+            sprintf(
+                "components/%s/configs/%s/metadata",
+                $options->getComponentConfiguration()->getComponentId(),
+                $options->getComponentConfiguration()->getConfigurationId()
+            ),
+            [
+                'metadata' => $options->getMetadata(),
+            ]
+        );
+    }
+
+    public function listConfigurationMetadata(ListConfigurationMetadataOptions $options)
```

Comment:
Tady mi přijde, že parametr by měl být taky ConfigurationMetadata jako u add, resp. nechápu, proč by to mělo být něco jiného